### PR TITLE
Fix dead links and wrong python script name

### LINKED ISSUE #41562: 404 on link in openstack dynamic inventory documentation
##### SUMMARY

Dead link in the [OpenStack Dynamic Inventory Documentation](https://docs.ansible.com/ansible/2.4/intro_dynamic_inventory.html#example-openstack-external-inventory-script)

##### ISSUE TYPE
 - Documentation Report

##### COMPONENT NAME

docs/docsite/rst/user_guide/intro_dynamic_inventory.rst

##### ANSIBLE VERSION

```
ansible 2.5.1
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.15rc1 (default, Apr 15 2018, 21:51:34) [GCC 7.3.0]
```

##### OS / ENVIRONMENT

N/A

##### STEPS TO REPRODUCE

```yaml

```

https://docs.ansible.com/ansible/2.4/intro_dynamic_inventory.html#example-openstack-external-inventory-script

##### EXPECTED RESULTS

Source code for a python script

##### ACTUAL RESULTS

404 not found

### PR DESCRIPTION
**Pull request by czunker**
Replace link to OpenStack Dynamic Inventory with current name of the script.
Change name of script to prevent Python module import errors.

Fixes #41562

##### SUMMARY

Fix dead link. The name of the script has changed.

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

docs/docsite/rst/user_guide/intro_dynamic_inventory.rst

##### ANSIBLE VERSION

N/A

### CODE REVIEW

**Review by achinthagunasekara — APPROVED** [review]

**Comment by s-hertel** [comment]
Looks like this is a duplicate of #41459 plus an additional note about the conflicting name. Leaving open so the note can be merged too after 41459 is.

**Comment by acozine** [comment]
Note was added to #41459 before we merged it. Closing this PR since all changes have been merged and backported to `stable-2.6` (#43455).

**Pull request closed without merging**
